RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 * [*] [internal] Change how a post is fetched when selecting a post in Spotlight search result. [#21439]
 * [*] [internal] Change how a post's revision is fetched from the post history screen. [#21440]
 * [**] [internal] Replace the progress indicator implementation in uploading featured image from "Post Settings" [#21438]
+* [***] [Jetpack-only] Contact Support: Need assistance? Use Jetpack mobile assistant for immediate help. Create a support ticket for further assistance. [#21467]
```

Comment:
I'm not sure if this is descriptive enough for someone unfamiliar with the feature to know what it is (e.g. the editorial team or folks testing the beta).

May I suggest an alternative? The same suggestion applies to Android @ravishanker.

> Contact Support: Add a new chat-based support channel where users can get answers from a bot trained to help app users. Users can still create a support ticket to talk to a Happiness Engineer if they don't find the answer they're looking for [#21467]

Reply:
Some additional context: pdcxQM-Km-p2

Reply:
Thanks! I agree your suggestion sounds more descriptive of what the feature actually does. I would accept your suggestion 👍

Reply:
I agree with the suggestion and updated the Android release notes.: https://github.com/wordpress-mobile/WordPress-Android/pull/19124

Reply:
Thanks folks!